src/Symfony/Component/Form/Form.php

```diff
@@ -63,23 +64,23 @@
 class Form implements \IteratorAggregate, FormInterface
 {
     /**
-     * The form's configuration.
+     * The form's configuration instance.
      *
      * @var FormConfigInterface
      */
     private $config;
 
     /**
-     * The parent of this form.
+     * The parent of this form or null.
```

Comment:
For me it’s enough to reflect it in the return php doc

Reply:
Another way is to remove it and keep `@param` only :)

Reply:
But I think both should be consistent.